packages/mask/src/plugins/Predict/Azuro/components/RedeemButton.tsx

```diff
@@ -0,0 +1,57 @@
+import { Grid } from '@mui/material'
+import LoadingButton from '@mui/lab/LoadingButton'
+import { makeStyles } from '@masknet/theme'
+import { useI18N } from '../../../../utils/i18n-next-ui'
+import type { UserBet } from '../types'
+import { useAsyncFn } from 'react-use'
+import { useAzuroContract } from '../hooks/useAzuroContract'
+import { useAccount } from '@masknet/plugin-infra/web3'
+import { NetworkPluginID } from '@masknet/web3-shared-base'
+
+const useStyles = makeStyles()((theme) => ({
+    redeemButton: {
+        background: theme.palette.success.light,
+    },
+}))
+
+interface RedeemButtonProps {
+    bet: UserBet
+    retry: () => void
+}
+
+export function RedeemButton(props: RedeemButtonProps) {
+    const { t } = useI18N()
+    const { bet, retry } = props
+    const { classes } = useStyles()
+    const azuroContract = useAzuroContract()
+    const account = useAccount(NetworkPluginID.PLUGIN_EVM)
+
+    const [{ loading, error }, doFetch] = useAsyncFn(async () => {
+        const config = {
+            from: account,
+            gas: await azuroContract?.methods.withdrawPayout(bet.nftId).estimateGas({ from: account }),
+        }
+
+        const tx = await azuroContract?.methods.withdrawPayout(bet.nftId).send(config)
```

Comment:
Please use the connection to send a transaction.

```ts
const connection = useWeb3Connection(NetworkPluginID.PLUGIN_EVM)

useAsyncFn(async () => {
    const txHash = await connection.sendTransaction()
}, [connection])
```

Reply:
I'm working on this one

Reply:
Done.